README.md

```diff
@@ -11,11 +11,12 @@ This information can be updated quite often, but is all peer reviewed. If you se
 * [Feature Toggles](process/feature-toggles.md)
 
 
-## Coding Standards
+## [Coding Standards](/codingstandards)
```

Comment:
Do you think we should link headings, or instead, link each item under the heading?